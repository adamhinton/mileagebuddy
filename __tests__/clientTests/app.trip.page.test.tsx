import "@testing-library/jest-dom";
import { render } from "@testing-library/react";
import TestReduxStore from "../../src/app/utils/unitTestUtils/dummyReduxStore";
import TripCreationPage from "@/app/trip_planner/page";

describe("TripPage", () => {
	it("renders without errors", () => {
		render(
			<TestReduxStore>
				<TripCreationPage />
			</TestReduxStore>
		);
	});

	it("matches snapshot from 4.30.2025", () => {
		const { container } = render(
			<TestReduxStore>
				<TripCreationPage />
			</TestReduxStore>
		);
		expect(container).toMatchSnapshot();
	});
});
