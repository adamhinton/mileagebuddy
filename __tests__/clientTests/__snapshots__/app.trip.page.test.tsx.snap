// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TripPage matches snapshot from 4.30.2025 1`] = `
<div>
  <h1>
    Placeholder
  </h1>
</div>
`;
